hydrolib/core/io/ini/models.py

```diff
@@ -25,6 +25,7 @@
 from hydrolib.core.io.ini.util import (
     get_from_subclass_defaults,
```

Comment:
A general comment:
I think the cross sections should have their own namespace like roughness (friction) now has.

Reply:
Yes, I fully agree. And I will do that later in a separate issue, because this issue about unit tests was already work-in-progress --> #149.